.github/workflows/run_doublet-detection.yml

```
@@ -0,0 +1,62 @@
+# This is a workflow to run the doublet-detection module
+# This module has both an renv and a conda environment
+#
+# Analysis modules are run based on three triggers:
+# - Manual trigger
+# - On pull requests where code in the module has changed
+# - As a reusable workflow called from a separate workflow which periodically runs all modules
+
+name: Run doublet-detection analysis module
+env:
+  MODULE_PATH: analyses/doublet-detection
+  AWS_DEFAULT_REGION: us-east-1
+
+concurrency:
+  # only one run per branch at a time
+  group: doublet-detection-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - analyses/doublet-detection/**
+jobs:
+  run-module:
+    runs-on: ubuntu-latest
```

This is needed for conda environments to work properly
```suggestion
  run-module:
    runs-on: ubuntu-latest
    defaults:
      run:
        shell: bash -el {0}
```